<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recherche</class>
 <widget class="QDialog" name="Recherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Recherche">
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_projet">
       <property name="text">
        <string>Projet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_projet"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton_tache">
       <property name="text">
        <string>TÃ¢che</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_tache"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="radioButton_Interv">
       <property name="text">
        <string>Intervenant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Interv"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
